tests/Integration/inc/ThirdParty/Plugins/Ecommerce/WooCommerceSubscriber/showEmptyProductGalleryWithDelayJS.php

```diff
@@ -0,0 +1,81 @@
+<?php
+namespace WP_Rocket\Tests\Integration\inc\ThirdParty\Plugins\Ecommerce\WooCommerceSubscriber;
+
+use WP_Rocket\Engine\Optimization\DelayJS\HTML;
+use WP_Rocket\Tests\Integration\TestCase;
+use WP_Rocket\ThirdParty\Plugins\Ecommerce\WooCommerceSubscriber;
+use Mockery;
+use Brain\Monkey\Functions;
```

Comment:
```suggestion
use WP_Rocket\Tests\Integration\TestCase;
```

The others are not used in this file